### PR TITLE
feat: adding is_public field for session fields in custom from

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5100

organizer can decide which field should be shown public and which should be private.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/models/custom_form.py [hunk 1]
```diff
@@ -156,6 +156,7 @@ class TYPE:
     description = db.Column(db.String, nullable=True)
     is_required = db.Column(db.Boolean, default=False)
     is_included = db.Column(db.Boolean, default=False)
+    is_public = db.Column(db.Boolean, default=False)
```
Comment (iamareebjamal) [inline app/models/custom_form.py:159]: nullable False

File: migrations/versions/rev-2020-12-19-13:38:15-cd0e3edd52f9_.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = 'cd0e3edd52f9'
+down_revision = '73dad592fcd6'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('custom_forms', sa.Column('is_public', sa.Boolean(), nullable=True))
```
Comment (iamareebjamal) [inline migrations/versions/rev-2020-12-19-13:38:15-cd0e3edd52f9_.py:21]: `nullable=False` and `server_default='False'`

**Comment by maze-runnar** [comment]
https://github.com/fossasia/open-event-server/pull/7525

File: app/models/custom_form.py [hunk 3]
```diff
@@ -156,7 +156,7 @@ class TYPE:
     description = db.Column(db.String, nullable=True)
     is_required = db.Column(db.Boolean, default=False)
     is_included = db.Column(db.Boolean, default=False)
-    is_public = db.Column(db.Boolean, default=False)
+    is_public = db.Column(db.Boolean, nullable=False)
```
Comment (iamareebjamal) [inline app/models/custom_form.py:159]: Why removed default False?

**Pull request closed without merging**